ui_lizmap_popup.py

```diff
@@ -7,7 +7,8 @@
 #
 # WARNING! All changes made in this file will be lost!
```

Comment:
As mentioned here, all changes in this file will be lost after a regeneration using `pyuic5`. You need to fix in the UI file first.

And I proposed a few months ago to remove the use the compiled UI file in https://github.com/3liz/lizmap-plugin/pull/119 but I have no answer from the 3Liz team.

Reply:
Thank you @Gustry  for the answer.